autospec/specfiles.py

```diff
@@ -1495,8 +1528,8 @@ def write_meson_pattern(self):
 
         if config.config_opts['use_avx2']:
             self._write_strip('CFLAGS="$CFLAGS -m64 -march=haswell" CXXFLAGS="$CXXFLAGS -m64 -march=haswell " LDFLAGS="$LDFLAGS -m64 -march=haswell" '
-                              'meson --prefix /usr --libdir=/usr/lib64/haswell --buildtype=plain {0} '
-                              '{1} builddiravx2'.format(config.extra_configure, config.extra_configure64))
+                              'meson --prefix /usr --libdir=/usr/lib64/haswell --buildtype=plain'
+                              '{0} {1} builddiravx2'.format(config.extra_configure, config.extra_configure64))
```

Comment:
This shouldn't be in this change, please rebase.

Reply:
O gosh, forgot to check for new changes. I'll fix it.